wheel/pep425tags.py

```diff
@@ -105,17 +109,78 @@ def get_abi_tag():
     return abi
 
 
-def get_platform():
+def get_platform(archive_root):
     """Return our platform name 'win32', 'linux_x86_64'"""
     # XXX remove distutils dependency
-    result = distutils.util.get_platform().replace('.', '_').replace('-', '_')
+    result = distutils.util.get_platform()
+    if result.startswith("macosx") and archive_root is not None:
+        prefix, base_version, suffix = result.split('-')
```

Comment:
I think this code should be moved into its own function to improve code readability.

Reply:
I do not understand. This is begin of big if statement in which proper macos platform tag is calculated. 
You suggest to extract whole macosx platform tag calculation? Or something else?

Reply:
The macOS specific tag calculation code, yes. Might even make it easier to test on its own. Do you object?

Reply:
No. I was not sure what you suggest.